internal/oem/oem.go

```diff
@@ -79,6 +85,14 @@ func init() {
 		flags: map[string]string{
 			"online-timeout": "0",
 		},
+		config: types.Config{
+			Systemd: types.Systemd{
+				Units: []types.SystemdUnit{{
+					Name:   "coreos-metadata-sshkeys@core.service",
```

Comment:
This doesn't work due to a systemctl issue. We'll have to workaround it for now.